src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java

```diff
@@ -57,7 +69,33 @@ public void verifyHeaderBasicElements(SoftAssert softAssert) {
         softAssert.assertTrue(instagramButton.isElementPresent(), "Instagram button was not found!");
         softAssert.assertTrue(youTubeButton.isElementPresent(), "YouTube button was not found!");
         softAssert.assertTrue(rssButton.isElementPresent(), "RSS button was not found!");
-        softAssert.assertTrue(logInButton.isElementPresent(), "Log in button was not found!");
+        softAssert.assertTrue(logInIcon.isElementPresent(), "Log in button was not found!");
         softAssert.assertTrue(signUpButton.isElementPresent(), "Sign up button was not found!");
     }
+
+    public void openFieldsForLogIn() {
+        logInIcon.click();
+    }
+
+    public void typeEmail(String email) {
+        emailLabel.type(email);
+    }
+
+    public void typePassword(String password) {
+        passwordLabel.type(password);
+    }
+
+    public void logIn() {
+        String email = "s9rowa@mail.ru";
```

Comment:
Looks like it's constant values.According to Java conventions constant should be named in Upper case.